src/components/MainArea.js

```diff
@@ -60,18 +64,24 @@ const DeleteButton = styled.Button`
 
 class MainArea extends Component<Props, State> {
   state: State = {
-    title: this.props.note.title,
     editorState: EditorState.createWithContent(
-      ContentState.createFromBlockArray(
-        convertFromHTML(this.props.note.content),
-      ),
+      this.props.note.content
+        ? convertFromRaw(JSON.parse(this.props.note.content))
+        : ContentState.createFromText('start typing...'),
     ),
   }
 
-  onContentChange = newContent => {
+  onEditorChange = editorState => {
+    this.setState({ editorState: editorState })
+  }
+
+  onContentChange = () => {
```

Comment:
Is this handler called on every keystroke? If so, might be useful to debounce it so it doesn't slow down the app too much, but that can always be done later if the UI starts getting slow.